src/ja/user/message/query.py

```diff
@@ -259,5 +260,42 @@ def from_dict(cls, property_dict: Dict[str, object]) -> "QueryCommand":
                          special_resources_list, cpu_threads, memory, before, after)
         return a
 
-    def execute(self, database: ServerDatabase) -> "ServerResponse":
-        pass
+    def execute(self, database: ServerDatabase) -> Response:
+        jobs: List[DatabaseJobEntry] = database.query_jobs(None, -1, None)  # Query all DatabaseJobEntries
```

Comment:
I think you should try to pass as many arguments as possible to the query_jobs command, since SQL is faster than ourselves :)